adt7410/adt7410.go

```diff
@@ -0,0 +1,108 @@
+package adt7410
+
+import (
+	"machine"
+	"time"
+)
+
+const (
```

Comment:
This all could be in a `registers.go` file in the same package, to match the other i2c devices?